build.sbt

```diff
@@ -276,16 +276,16 @@ lazy val apacheSparkBenchmarks = (project in file("benchmarks/apache-spark"))
     // Override versions pulled in by dependencies.
     dependencyOverrides ++= Seq(
       // Force newer version of Hadoop (compatibility with IBM Semeru JDK builds).
-      "org.apache.hadoop" % "hadoop-client-runtime" % "3.3.6",
+      "org.apache.hadoop" % "hadoop-client-runtime" % "3.4.0",
       // Force common (newer) version of Netty packages.
       "io.netty" % "netty-all" % nettyVersion,
       // Force common (newer) version of Jackson packages.
       "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % jacksonVersion,
       "com.fasterxml.jackson.module" %% "jackson-module-scala" % jacksonVersion,
       // Force common (newer) version of Jersey packages.
-      "org.glassfish.jersey.containers" % "jersey-container-servlet" % jerseyVersion,
-      "org.glassfish.jersey.core" % "jersey-server" % jerseyVersion,
-      "org.glassfish.jersey.inject" % "jersey-hk2" % jerseyVersion,
+      "org.glassfish.jersey.containers" % "jersey-container-servlet" % jersey2Version,
+      "org.glassfish.jersey.core" % "jersey-server" % jersey2Version,
+      "org.glassfish.jersey.inject" % "jersey-hk2" % jersey2Version,
```

Comment:
There is probably no need to rename `jerseyVersion` to `jersey2Version`?

Reply:
I can revert it, but I updated everything, `4.0.0-M1` would have been newer than `2.43`, so I wanted to emphasize the compatibility 😉